Source/WebCore/page/OpportunisticTaskScheduler.cpp

```diff
@@ -98,10 +106,12 @@ void OpportunisticTaskScheduler::runLoopObserverFired()
         if (m_runloopCountAfterBeingScheduled > minimumRunloopCountWhenScheduledWorkIsImminent)
             return true;
 
+        dataLogLnIf(verbose, "GAVEUP: task does not get scheduled ", remainingTime, " ", hasImminentlyScheduledWork(), " ", page->preferredRenderingUpdateInterval(), " ", m_runloopCountAfterBeingScheduled);
```

Comment:
Interesting! We don't generally use dataLogs in WebCore and above, but it's probably fine for this use, given that this code is closely tied to the GC lifecycle.

I wonder if we could add something like `"[OPPORTUNISTIC TASK]"` to the beginning of all these datalog message, so that it's easier to filter by that when debugging/analyzing the OpTask lifecycle.

Reply:
`"[OPPORTUNISTIC TASK]"` sounds good!